RFC/src/RFC-0300_DAN.md

```diff
@@ -53,9 +53,9 @@ The goal of this RFC is to describe the key features of the Tari second layer, a
 ## Related RFCs
 
 * [RFC-0100: Base layer](RFC-0100_BaseLayer.md)
-* [RFC-0310: Digital assets](RFC-0310_AssetTemplates.md)
-* [RFC-0311: Second layer consensus strategies](RFC-3011_SecondLayerConsensus.md)
-* [RFC-0312: Validator nodes](RFC-3012_ValidatorNodes.md)
+* [RFC-0311: Digital assets](RFC-0311_AssetTemplates.md)
```

Comment:
Let's not change filenames / RFC numbers once they're in draft format -- there are links to them out on the internet now. I guess Asset templates hasn't been merged in yet, are any of these affected?

Reply:
So all of those files were placeholders at this stage, no actual content in any of the ones that got updated?

Reply:
Cool